core/src/main/java/org/apache/calcite/tools/RelBuilder.java

```diff
@@ -667,19 +668,32 @@ public AggCall max(String alias, RexNode operand) {
    *
    * <p>Returns this builder.
    *
-   * @param tableName Name of table
+   * @param tableNames Name of table (can optionally be fully qualified)
    */
```

Comment:
can optionally be qualified

(fully would be database.catalog.schema.table, but schema.table is ok too)